updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -13,19 +13,39 @@ You must mirror container images onto a mirror registry before you can update a
 Your mirror registry must be running at all times while the cluster is running.
 ====
 
-There are two methods for mirroring images onto a mirror registry:
+The following steps outline the high-level workflow on how to mirror images to a mirror registry:
 
-* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-ocp-resources-ocmirror[Using the oc-mirror OpenShift CLI (`oc`) plugin]
+. Install the OpenShift CLI (`oc`) on all devices being used to retrieve and push release images.
 
-* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#update-mirror-repository-adm-release-mirror_mirroring-ocp-image-repository[Using the `oc adm release mirror` command]
+. Download the registry pull secret and add it to your cluster.
 
-Compared to using the `oc adm release mirror`command, the oc-mirror plugin has the following advantages:
+. If you use the xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-ocp-resources-ocmirror[oc-mirror OpenShift CLI (`oc`) plugin]:
+
+.. Install the oc-mirror OpenShift CLI plugin on all devices being used to retrieve and push release images.
```

Comment:
[Above](https://github.com/openshift/openshift-docs/pull/58739/files#diff-a504b4a4e4d8ecfc7c9d4c6c0024d18c39dee47d9f53a60c5190841f9a2372f3R12) you used _oc-mirror plugin_ on a second reference. Do you want to be consistent?